lib/rucio/common/schema/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright European Organization for Nuclear Research (CERN)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+from ConfigParser import NoOptionError, NoSectionError
+
+from rucio.common import config
+
+if config.config_has_section('policy'):
+    try:
+        POLICY = config.config_get('policy', 'schema')
+    except (NoOptionError, NoSectionError) as error:
+        POLICY = 'generic'
+else:
+    POLICY = 'generic'
+
+if POLICY.lower() == 'generic':
+    from .generic import *  # NOQA pylint:disable=wildcard-import
```

Comment:
pylint complains for me on all the relative imports with W0403, better to do full imports

Reply:
weird!
pylint lib/rucio/common/schema/__init__.py- --> Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
and no complains.

$ pylint --version
pylint 1.7.5, 
astroid 1.5.3
Python 2.7.10 (default, Jul 15 2017, 17:16:57) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]